test/options_test.dart

```diff
@@ -213,57 +181,57 @@ An example in an unusual dir.
 '''),
         ]),
       ],
-      resourceProvider: resourceProvider,
     );
-    final packageGraph =
-        await (await createPackageBuilder()).buildPackageGraph();
+    final packageGraph = await bootBasicPackage(
+      packagePath,
+      packageMetaProvider,
+      packageConfigProvider,
+    );
     final classFoo = packageGraph.allLocalModelElements
         .where((e) => e.isCanonical)
         .whereType<Class>()
         .firstWhere((c) => c.name == 'Foo');
     expect(classFoo.documentationAsHtml,
         contains('<code class="language-dart">An example in an unusual dir.'));
-  });
+  }
 
-  test("'format=md' option generates markdown files", () async {
-    packagePath = await d.createPackage(
+  void test_formatEqualMdOptionGeneratesMarkdownFiles() async {
```

Comment:
Can we name these more like `test_formatOption_md_generatesMarkdownFiles` and the one below as `test_formatOption_bad_dartDocFailure` and the others in a similar fashion where the first test after `_` is the option being tested? It's easier to read and categorize :D

Reply:
Thanks, great catch! I added some underscores to other test cases after the word `Option`; nice separation.